### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by linupychiang**
FastAPI文档index部分中文翻译

### CODE REVIEW

**Review by phy25 — CHANGES_REQUESTED** [review]
I don't think this is the proper way to merge translations into master by removing all the original sentences.

**Comment by sm-Fifteen** [comment]
I don't even know if the documentation even supports having translations with the way is is setup right now; not to mention that translating all of it would be a huge endeavor. I know FastAPI has contributors from a variety of countries speaking a variety of languages, but I guess English has done a good enough job at getting everyone involved to understand each other so far, so it's not a priority for anyone at the moment.

File: docs/index.md [hunk 1]
```diff
@@ -1,110 +1,91 @@
...
-"*I’m over the moon excited about **FastAPI**. It’s so fun!*"
+我对FastAPI感到欣喜若狂。太有趣了！
 
-<div style="text-align: right; margin-right: 10%;">Brian Okken - <strong><a href="https://pythonbytes.fm/episodes/show/123/time-to-right-the-py-wrongs?time_in_sec=855" target="_blank">Python Bytes</a> podcast host</strong> <a href="https://twitter.com/brianokken/status/1112220079972728832" target="_blank"><small>(ref)</small></a></div>
+<p align='right'>——Brian Okken - Python Bytes podcast host</p>
 
 ---
 
-"*Honestly, what you've built looks super solid and polished. In many ways, it's what I wanted **Hug** to be - it's really inspiring to see someone build that.*"
+老实说，你做的东西看起来很可靠和漂亮。从很多方面来说，这就是我希望**Hug**成为的样子。看到有人这么做，真的很鼓舞人心。
```
Comment (juhovh-aiven) [inline docs/index.md]: 看到有人这么做 doesn't convey very well the point of someone building what Hug was supposed to be.

File: docs/index.md [hunk 2]
```diff
@@ -120,12 +101,9 @@ def read_item(item_id: int, q: str = None):
     return {"item_id": item_id, "q": q}
 ```
 
-<details markdown="1">
-<summary>Or use <code>async def</code>...</summary>
+如果您的代码使用了`async` / `await`，请使用 `async def` ：
```
Comment (juhovh-aiven) [inline docs/index.md]: Not sure why the past tense in the former part of the sentence.

File: docs/index.md [hunk 3]
```diff
@@ -141,71 +119,54 @@ async def read_item(item_id: int, q: str = None):
...
+![Swagger UI](http://linupyoos.oss-cn-beijing.aliyuncs.com/blogs/2020-02-29-134300.png)
 
-![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+### 另一个API文档
 
-### Alternative API docs
+现在，切换到[http://127.0.0.1:8000/redoc](http://127.0.0.1:8000/redoc)路径，您将看到另一个自动文档(由[ReDoc](https://github.com/Rebilly/ReDoc)提供)
 
-And now, go to <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>.
+![ReDoc](http://linupyoos.oss-cn-beijing.aliyuncs.com/blogs/2020-02-29-135552.png)
```
Comment (juhovh-aiven) [inline docs/index.md]: More external images.

**Review by ghost — COMMENTED** [review]
Didn't do a full review, but there are some inaccuracies, many references to 3rd party images that are not guaranteed to be immutable, many of the helpful additional comments have been removed. Most importantly, rewriting the index.md is not ideal, (IMHO) this translation is probably better left in a forked repo for other people to see if necessary.

File: docs/index.md [hunk 4]
```diff
@@ -1,110 +1,91 @@
...
-    <img src="https://codecov.io/gh/tiangolo/fastapi/branch/master/graph/badge.svg" alt="Coverage">
-</a>
-<a href="https://pypi.org/project/fastapi" target="_blank">
-    <img src="https://badge.fury.io/py/fastapi.svg" alt="Package version">
-</a>
-<a href="https://gitter.im/tiangolo/fastapi?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge" target="_blank">
-    <img src="https://badges.gitter.im/tiangolo/fastapi.svg" alt="Join the chat at https://gitter.im/tiangolo/fastapi">
-</a>
-</p>
+![FastAPI](http://linupyoos.oss-cn-beijing.aliyuncs.com/blogs/2020-02-29-114916.png)
```
[the remark this answers is not in the document]
Comment (linupychiang) [reply docs/index.md]: Thanks for your comments.  Recently, I learned a lot about Github.

[the remark this answers is not in the document]
Comment (linupychiang) [reply docs/index.md]: Thanks for your comments.  Recently, I learned a lot about Github.

**Comment by tiangolo** [comment]
Thanks for your effort and contribution @linupychiang ! :cake: :rocket: And thanks a lot @juhovh-aiven for helping with the review ...and for keeping the review in English (that way I can understand the conversation :sweat_smile: ).

@linupychiang I finally finished the set up to have translations for the docs, here are the new docs: https://fastapi.tiangolo.com/contributing/#translations

You can probably still copy a lot of the text you had already translated, as the file structure is the same.

**Pull request closed without merging**